### ISSUE TITLE
Obsolete paragraph regarding versioning in Webpack Encore

### ISSUE THREAD

**Issue by royklutman** [symfony/symfony-docs#10176]
Since Webpack Encore now generates a `config/packages/assets.yaml` file containing the settings needed to load assets from the `manifest.json` file ([reference](https://github.com/symfony/recipes/blob/master/symfony/webpack-encore-pack/1.0/config/packages/assets.yaml)):
```yaml
# config/packages/assets.yaml
framework:
    assets:
        json_manifest_path: '%kernel.project_dir%/public/build/manifest.json'
```
You don't actually have to manually add `json_manifest_path` to `config/packages/framework.yaml` anymore, so the following paragraph becomes obsolete: http://symfony.com/doc/current/frontend/encore/versioning.html#loading-assets-from-the-manifest-json-file

I wouldn't mind creating a PR to remove or edit the paragraph myself, but my english is subpar sadly.

**Comment by weaverryan** [comment]
You're totally right! And we do assume you're using Flex for the 4.0 branch and higher. Your English seems actually pretty great to me :). If you did open a PR, if there were any English tweaks needed, we can make them easily - opening the PR and thinking about how the information should be organized is the hard part :).

**Comment by javiereguiluz** [comment]
Fixed by #10190.
